cli/src/tunnels/control_server.rs

```diff
@@ -74,7 +67,7 @@ struct HandlerContext {
 	/// Connected VS Code Server
 	code_server: CodeServerCell,
 	/// Potentially many "websocket" connections to client
-	server_bridges: ServerBridgeListLock,
```

Comment:
Some abstraction so that the new multiplexer could be used in WSL control server as well.